packages/distribution/addon/abilities/inquiry.js

```diff
@@ -59,7 +59,8 @@ export default class InquiryAbility extends Ability {
       this.hasBasePermission &&
       // Since editing in the status ready has the same character as sending an
       // inquiry, we need to make sure that permission would be given
-      (this.isSuspended ||
+      ((this.isSuspended &&
+        this.hasCustomPermission("editInquiry", this.model)) ||
         (this.isReady &&
           this.hasCustomPermission("sendInquiry", this.model))) &&
```

Comment:
If we have a custom permission for it now we should drop the send inquiry custom permission logic in here.

Reply:
Couldn't the permission still be variable depending on the status?

Reply:
If it has to be it can be handled in the custom permission

Reply:
Fair point, I updated it.